data/rulesets.js

```diff
@@ -372,16 +372,39 @@ exports.BattleFormats = {
 		},
 		onValidateTeam: function (team, format) {
 			let abilityTable = {};
+			let clones = {
+				airlock: ['cloud9'],
+				battlearmor: ['shellarmor'],
+				clearbody: ['whitesmoke'],
+				cloud9: ['airlock'],
+				dazzling: ['queenlymajesty'],
+				insomnia: ['vitalspirit'],
+				moldbreaker: ['teravolt', 'turboblaze'],
+				powerofalchemy: ['receiver'],
+				queenlymajesty: ['dazzling'],
+				receiver: ['powerofalchemy'],
+				shellarmor: ['battlearmor'],
+				teravolt: ['moldbreaker', 'turboblaze'],
+				turboblaze: ['moldbreaker', 'teravolt'],
+				vitalspirit: ['insomnia'],
+				whitesmoke: ['clearbody'],
+			};
```

Comment:
also rough skin and iron barbs, wimp out and emergency exit, plus and minus, gooey and tangling hair

they might not be viable abilities, but they're still dupes